src/elements/content-uploader/ContentUploader.tsx

```diff
@@ -289,9 +294,8 @@ class ContentUploader extends Component<ContentUploaderProps, State> {
      */
     getNewFiles = (files: Array<UploadFileWithAPIOptions | File>): Array<UploadFileWithAPIOptions | File> => {
         const { rootFolderId } = this.props;
-        const { itemIds } = this.state;
```

Comment:
is `itemIds` used anywhere now?

Reply:
No its not. But to reduce risk I decided to leave it in for now. I can create a followup ticket to remove it